pyproject.toml

```diff
@@ -46,7 +46,7 @@ gremlinpython = "^3.5.2"
 backoff = ">=2.0.0,<3.0.0"
 SPARQLWrapper = { version = "^2.0.0", optional = true }
 pyodbc = { version = "~4.0.32", optional = true }
-modin = { version = "~0.16.0", optional = true }
+modin = { version = "~0.18.0", optional = true }
```

Comment:
Same issue than with Ray, we need to make sure that Glue can support this version first...